pandas/tests/io/test_common.py

```diff
@@ -129,7 +129,7 @@ def test_iterator(self):
             (pd.read_csv, "os", FileNotFoundError, "csv"),
             (pd.read_fwf, "os", FileNotFoundError, "txt"),
             (pd.read_excel, "xlrd", FileNotFoundError, "xlsx"),
-            (pd.read_feather, "feather", Exception, "feather"),
+            (pd.read_feather, "pyarrow", Exception, "feather"),
```

Comment:
can this be ArrowIOError?

Reply:
ArrowIOError [has existed since 0.3](https://github.com/apache/arrow/commit/2aed7845fbc9e3d91ab9d16965ee9f6f3abc668b), originally as a subclass of IOError and [now as an alias of it](https://github.com/apache/arrow/commit/28f4980a15b3c342e9d8866404d3c5f222efed29#diff-29a4597facbb7bccbed5dbea777e58b2R61), and we require >= 0.13 so yes.

IOError is easier than ArrowIOError since we can't be sure of importing pyarrow at this point.

Reply:
sure that's fine (just try to avoid the bare Exception)